doc/managing_pachyderm/general_troubleshooting.md

```diff
@@ -0,0 +1,227 @@
+
+It's possible that the connection is just taking a while. Occasionally this can happen if your cluster is far away (deployed in a region across the country). Check your internet connection.
+
+It's also possible that you haven't poked a hole in the firewall to access the node on this port. Usually to do that you adjust a security rule (in AWS parlance a security group). For example, on AWS, if you find your node in the web console and click on it, you should see a link to the associated security group. Inspect that group. There should be a way to "add a rule" to the group. You'll want to enable TCP access (ingress) on port 30650. You'll usually be asked which incoming IPs should be whitelisted. You can choose to use your own, or enable it for everyone (0.0.0.0/0).
+
+
+### Certificate Error When Using Kubectl
+
+#### Symptom
+
+This can happen on any request using `kubectl` (e.g. `kubectl get all`), but it can also be seen when running `pachctl port-forward` because it uses `kubectl` under the hood. In particular you'll see:
+
+```
+$ kubectl version
+Client Version: version.Info{Major:"1", Minor:"6", GitVersion:"v1.6.4", GitCommit:"d6f433224538d4f9ca2f7ae19b252e6fcb66a3ae", GitTreeState:"clean", BuildDate:"2017-05-19T20:41:24Z", GoVersion:"go1.8.1", Compiler:"gc", Platform:"darwin/amd64"}
+Unable to connect to the server: x509: certificate signed by unknown authority
+```
+
+#### Recourse
+
+Check if you're on any sort of VPN or other egress proxy that would break SSL.
+
+### Uploads/Downloads are Slow
+
+#### Symptom
+
+Any `pachctl put-file` or `pachctl get-file` commands are slow.
+
+#### Recourse
+
+Check if you're using port-forwarding. Port forwarding throttles traffic to ~1MB/s. If you need to do large downloads/uploads you should consider using the `ADDRESS` variable instead to connect directly to your k8s master node. [See this note](./getting_started/other_installation.html?highlight=ADDRESS#usage)
+
+You'll also want to make sure you've allowed ingress access through any firewalls to your k8s cluster.
+
+---
+
+## Problems Running Pipelines
+
+### All your pods / jobs get evicted
+
+#### Symptom
+
+Running,
+
+```
+$ kubectl get all
+```
+
+shows a bunch of pods that are marked `Evicted`. If you `kubectl describe ...` one of those evicted pods, you see an error saying that it was evicted due to disk pressure.
+
+
+#### Recourse
+
+Your nodes are not configured with a big enough root volume size.  You need to make sure that each node's root volume is big enough to store the biggest datum you expect to process anywhere on your DAG plus the size of the output files that will be written for that datum.
+
+Let's say you have a repo with 100 folders. You have a single pipeline with this repo as an input, and the glob pattern is `/*`. That means each folder will be processed as a single datum. If the biggest folder is 50GB and your pipeline's output is about 3 times as big, then your root volume size needs to be bigger than:
```

Comment:
This advice should probably go in a section of the docs that's proactive in addition to here (where we're reacting to the problem). Maybe under the data management section? Basically they should plan to deploy their cluster with enough root disk size for all inputs/outputs of all simultaneous datums that will be running. (And we can provide some math there ... e.g. 10 pipelines x 20 datums each x 1GB per datum => 200GB+ volume size is needed ... and other examples for lazy pipelines / etc). If you want to do that as a separate PR, that's cool, but let's make a GH issue so we don't forget.